libc/utils/docgen/signal.json

```diff
@@ -0,0 +1,21 @@
+{
+  "macros": [
+    "__STDC_VERSION_FENV_H__"
+  ],
+  "functions": {
+    "kill": null,
+    "sigaction": null,
+    "sigaddset": null,
+    "sigaltstack": null,
+    "sigdelset": null,
+    "sigemptyset": null,
+    "sigfillset": null,
+    "sigprocmask": null,
```

Comment:
So I was thinking about these POSIX defined functions (none of this needs to be solved in this PR, but is perhaps worth thinking about how we can do something better for POSIX functions in the future).

It is possible to create a hyperlink to a page about them, example: https://pubs.opengroup.org/onlinepubs/009604499/functions/kill.html

Perhaps we either add more specialized handling of the 'defined' field to support URLs, or use a different field. IDK yet.